charts/fleet-agent/100.0.3+up0.3.9/Chart.yaml

```diff
@@ -1,9 +1,11 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
release in 2.6.4 based on https://github.com/rancher/rancher/issues/36185, this does not look correct.

Tested versions are 1.18-1.22, not ending at 1.21, based on the support matrix in https://www.suse.com/assets/EN-Rancherv2.6.4-150422-0151-56.pdf, but 1.16-1.17 are fine to keep in the range.

**Needs changes**: kube-version for all charts released in 2.6.4+ are too narrow in range since they should include 1.22